components/centraldashboard/app/api.ts

```diff
@@ -27,6 +32,16 @@ export class Api {
     return this.platformInfo;
   }
 
+  private getProfileAndAuth(req: express.Request): [DefaultApi, AuthObject] {
```

Comment:
Do you think maybe this makes sense to separate into two methods? Instantiating the controller from the host name of the request seems like one logical function and resolving the authentication header seems like another. I could imagine that the latter will eventually need to expand as additional authentication methods are supported. In addition, the return of two unrelated objects in a tuple seems awkward.

Reply:
Fixed, and updated, no longer need to do this